tools/bazel/bazel8.bazelrc

```diff
@@ -0,0 +1,140 @@
+import %workspace%/.aspect/bazelrc/bazel7.bazelrc
+
+build --remote_build_event_upload=minimal
+build --noexperimental_check_external_repository_files
+build --extra_toolchains=@graalvm//:bootstrap_runtime_toolchain
+
+# Compat: We keep the `bzlmod` config profile because it is injected for pre-Bazel 8 in CI.
+build --enable_bzlmod
+build:bzlmod --enable_bzlmod
+
+build:dev --lockfile_mode=update
+
+build:labs --experimental_worker_sandbox_hardening
+build:labs --experimental_worker_for_repo_fetching=platform
+build:labs --experimental_worker_multiplex_sandboxing
+build:labs --experimental_bzl_visibility
+
+build:strict --incompatible_depset_for_java_output_source_jars
+build:strict --incompatible_disallow_sdk_frameworks_attributes
+build:strict --incompatible_objc_alwayslink_by_default
+build:strict --incompatible_python_disable_py2
+build:strict --incompatible_always_include_files_in_data
+build:strict --incompatible_check_sharding_support
+build:strict --incompatible_check_testonly_for_output_files
+build:strict --incompatible_disable_starlark_host_transitions
+build:strict --incompatible_fix_package_group_reporoot_syntax
+build:strict --incompatible_package_group_has_public_syntax
+build:strict --incompatible_remote_use_new_exit_code_for_lost_inputs
+build:strict --incompatible_sandbox_hermetic_tmp
+build:strict --incompatible_unambiguous_label_stringification
+
+# ---- Inlined `performance.bazelrc` with modifications for Bazel 8
+
+# Speed up all builds by not checking if output files have been modified. Lets you make changes to
```

Comment:
@alexeagle I had to inline `performance.bazelrc` and `correctness.bazelrc` to get Bazel 8 working; I presume you already know about this, but an update to the Aspect configs didn't seem to help (they updated, but flags remained present which caused errors, and I can't fix that, because it will fail the Aspect config tests.)

Reply:
yup thanks, someone was just contributing to these on bazel-lib